pennylane/tape/qscript.py

```diff
@@ -375,6 +378,9 @@ def _update_circuit_info(self):
         ]
         self.is_sampled = any(is_sample_type)
         self.all_sampled = all(is_sample_type)
+        self.samples_computational_basis = any(
+            o.samples_computational_basis for o in self.measurements
+        )
```

Comment:
Should we add it as a property? This way the user can't modify its value.

Doing this we also remove the need of adding:
```python
new_qscript.samples_computational_basis = self.samples_computational_basis
```
in the `_update_par_info` method

Reply:
Sure! Changed it.